dev/source/docs/ros2-pi.rst

```diff
@@ -95,3 +95,183 @@ The simplest solution is to wire an open telemetry port.
 
 .. figure:: ../images/PiZero2WTelem.png
    :target: ../_images/PiZero2WTelem.png
+
```

Comment:
Need to tell the user to set up the correct SERIALn_FUNCTION to DDS. And that they need a custom build with DDS enabled.